atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -53,6 +54,12 @@ public final class OracleDdlTable implements DbDdlTable {
     private static final SafeLogger log = SafeLoggerFactory.get(OracleDdlTable.class);
     private static final String MIN_ORACLE_VERSION = "11.2.0.2.3";
 
+    @VisibleForTesting
+    static final String MISSING_OVERFLOW_TABLE_MESSAGE =
```

Comment:
Perhaps there's a counter point to this (as there always is in Atlas), but we _generally_ inline the message and copy it in the tests.

Reply:
Sounds good